assignment-final.Rmd

```diff
@@ -1,5 +1,39 @@
 ---
-title: "Final assignment"
+title: "Final assignment: Scientific report"
+output:
+    html_document:
+        toc: false
 ---
 
-> Uploaded when it is handed out.
+The final assignment is fairly straightforward, but also challenging. You, as a
+team, must write up a scientific report/manuscript. The best way to think about 
+this assignment is to imagine you are a scientist and you are writing a paper
```

Comment:
~The best way to think about this assignment is to i~Imagine you are scientists ~and you are~ writing a paper to publish your results in a peer reviewed journal.